docs/naming.md

```diff
@@ -31,56 +49,57 @@ The path is restricted to a limited set of characters (a-z, A-Z, 0-9, -, \_ ) an
 
 ### Macrons
 
-When a name contains macrons "Ōtorohanga" the macron is removed from the path but retained in the metadata's Title / Description.
+When a name contains macrons "Ōtorohanga" the macron is removed from the path but retained in the STAC Collection's Title / Description.
 
 ### Apostrophes
 
-Where a name contains an apostrophe "Hawke's Bay" the apostrophe is removed from the path but retained in the metadata's Title / Description.
+Where a name contains an apostrophe "Hawke's Bay" the apostrophe is removed from the path but retained in the STAC Collection's Title / Description.
 
 ### Path components
 
 #### `region`
 
 Is taken from a list of regions
 
-- New Zealand `new-zealand`
 - Northland `northland`
 - Auckland `auckland`
 - Waikato `waikato`
 - Bay of Plenty `bay-of-plenty`
 - Gisborne `gisborne`
+- Hawke's Bay `hawkes-bay`
 - Taranaki `taranaki`
-- Manawatu-Whanganui `manawatu-whanganui`
+- Manawatū-Whanganui `manawatu-whanganui`
 - Wellington `wellington`
-- Nelson `nelson`
-- Tasman `tasman`
-- Marlborough `marlborough`
-- Canterbury `canterbury`
 - West Coast `west-coast`
+- Canterbury `canterbury`
 - Otago `otago`
 - Southland `southland`
-- Offshore Islands `offshore-islands`
-- Pacific Islands `pacific-islands`
+- Tasman `tasman`
+- Nelson `nelson`
+- Marlborough `marlborough`
+- New Zealand `new-zealand`
 - Antarctica `antarctica`
+- Pacific Islands `pacific-islands`
+- Global `global`
```

Comment:
Are these in any particular order? If not, should they be alphabetical?

Reply:
Changed to alphabetical in https://github.com/linz/imagery/pull/336/commits/4365555